src/bindings/writeTypescript.ts

```diff
@@ -352,7 +352,12 @@ export function writeTypescript(
                                 r.message.text !== null &&
                                 r.message.text !== undefined
                             ) {
-                                receivers.push(`'${r.message.text}'`);
+                                // since we are using single quotes, escape them in the message name
+                                const escaped = r.message.text.replace(
```

Comment:
Could there also be a `\` in `r.message.text`?

Probably `JSON.stringify(escaped)` is what we need here.

Reply:
AFAIK, `JSON.stringify` doesn't escape `'` in string 🤔

Reply:
> Could there also be a \ in r.message.text?

Only a double one, like `\\`, https://github.com/tact-lang/tact/blob/229ce26cf95023ceff3d754d2622501d70b83cb6/src/grammar/grammar.ohm#L340

Reply:
also:
```
TactParseError: test/main.tact:15:17: Parse error: expected "\""

Line 15, col 17:
  14 | 
> 15 |     receive("get\'") {
                       ^
  16 | 
```

Reply:
But maybe we can just use:
```js
`string`
```

Reply:
> doesn't escape ' in string

That wouldn't be a problem if it was a `"`-string, would it? :D

> Only a double one, like `\\`

Then it probably shouldn't matter. It's just that serializing JS-compatible string is easily done with `JSON.stringify`, and there isn't really any way it can go wrong (JS-syntax-wise).

Reply:
> Then it probably shouldn't matter. It's just that serializing JS-compatible string is easily done with `JSON.stringify`, and there isn't really any way it can go wrong (JS-syntax-wise). If the goal is to generate TS in a way that when it runs runtime has the same exact string, `stringify` is the best option.

Now I understand, this seems like the best solution of all